ui/png-icons/src/main/res/drawable/chart_box_outline.xml

```diff
@@ -0,0 +1,8 @@
+<!-- drawable/chart_box_outline.xml -->
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:height="24dp"
+    android:width="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24">
+    <path android:fillColor="#FFFFFFFF" android:pathData="M9 17H7V10H9V17M13 17H11V7H13V17M17 17H15V13H17V17M19 19H5V5H19V19.1M19 3H5C3.9 3 3 3.9 3 5V19C3 20.1 3.9 21 5 21H19C20.1 21 21 20.1 21 19V5C21 3.9 20.1 3 19 3Z" />
```

Comment:
Isn't `#FFFFFFFF` invisible in the light theme? `?attr/action_icon_color` automatically switches the color between light and dark theme. See `ic_refresh`, for example.